openedx/core/djangoapps/plugins/utils.py

```diff
@@ -10,3 +16,32 @@ def get_module_path(app_config, plugin_config, plugin_cls):
         package_path=app_config.name,
         module_path=plugin_config.get(plugin_cls.RELATIVE_PATH, plugin_cls.DEFAULT_RELATIVE_PATH),
     )
+
+
+def parse_and_import_module_attr(attr_path):
```

Comment:
Django has a utility function for this: https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.module_loading.import_string

Reply:
ah.. there you are.  I was looking in the wrong place.  Thanks!  updated the code.